language/he/acp/groups.php

```diff
@@ -58,98 +58,100 @@
 	'GROUP_COLOR'					=> 'צבע הקבוצה',
 	'GROUP_COLOR_EXPLAIN'			=> 'הגדר את צבע שמות המשתמש של חברי הקבוצה, השאר ריק להגדרות הרגילות.',
 	'GROUP_CONFIRM_ADD_USERS'		=> array(
```

Comment:
Please note that some strings on this file were reverted back due to unknown reason. It should be safe to re-merge them.